### PR TITLE
Backport includeEndStreamAction in DeltasharingService to branch-0.7

### PR DESCRIPTION
**Pull request by linzhou-db**
Backport includeEndStreamAction in DeltasharingService to branch-0.7

### CODE REVIEW

**Comment by chakankardb** [comment]
Question @linzhou-db : Should we soak it for some time before we backport?

File: spark/src/main/scala/io/delta/sharing/spark/DeltaSharingClient.scala [hunk 1]
```diff
@@ -103,14 +114,16 @@ private[spark] class DeltaSharingRestClient(
     numRetries: Int = 10,
     maxRetryDuration: Long = Long.MaxValue,
     sslTrustAll: Boolean = false,
-    forStreaming: Boolean = false
+    forStreaming: Boolean = false,
+    endStreamActionEnabled: Boolean = false
```
Comment (linzhou-db) [inline spark/src/main/scala/io/delta/sharing/spark/DeltaSharingClient.scala:118]: @chakankardb this is changed to false.

File: spark/src/main/scala/io/delta/sharing/spark/util/ConfUtils.scala [hunk 2]
```diff
@@ -30,6 +30,9 @@ object ConfUtils {
   val MAX_RETRY_DURATION_CONF = "spark.delta.sharing.network.maxRetryDuration"
   val MAX_RETRY_DURATION_DEFAULT_MILLIS = 10L * 60L* 1000L /* 10 mins */
 
+  val INCLUDE_END_STREAM_ACTION_CONF = "spark.delta.sharing.query.includeEndStreamAction"
+  val INCLUDE_END_STREAM_ACTION_DEFAULT = "false"
```
Comment (linzhou-db) [inline spark/src/main/scala/io/delta/sharing/spark/util/ConfUtils.scala:34]: @chakankardb this is changed to false.

**Review by chakankardb — APPROVED** [review]

**Comment by linzhou-db** [comment]
local tests passed, merging

**Pull request merged**